src/Microsoft.Health.Dicom.Core/Features/Query/QueryLimit.cs

```diff
@@ -67,6 +102,7 @@ internal static class QueryLimit
             { QueryResource.StudySeries, ImmutableHashSet.Create(QueryTagLevel.Series)},
             { QueryResource.StudyInstances,  ImmutableHashSet.Create(QueryTagLevel.Series, QueryTagLevel.Instance) },
             { QueryResource.StudySeriesInstances,  ImmutableHashSet.Create(QueryTagLevel.Instance) },
+            { QueryResource.WorkitemInstances,  ImmutableHashSet.Create(QueryTagLevel.Instance) },
```

Comment:
Can we create a separate mapping workItems. We should not merge them.

Reply:
Removed as per the discussion